spec/project_spec.rb

```diff
@@ -192,9 +192,7 @@
 
   describe 'requiring all of `lib` with verbose warnings enabled' do
     it 'emits no warnings' do
-      allowed = lambda do |line|
-        line =~ /warning: private attribute\?$/ && RUBY_VERSION < '2.3'
-      end
+      allowed = ->(line) { line =~ /warning: private attribute\?$/ }
```

Comment:
I didn't place the `RUBY_VERSION` check at the right place in the PR where I added it. The idea is that now we don't need any exceptions, so we can fully removing the `allowed` proc, and the filtering below using it.

Reply:
I get it. I remembered the following issue. Thanks always!
https://bugs.ruby-lang.org/issues/10967